app/components/autocomplete/at_mention_group/index.js

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {connect} from 'react-redux';
+
+import {getTheme} from '@mm-redux/selectors/entities/preferences';
+
+import GroupMentionItem from './at_mention_group';
+
+function mapStateToProps(state, ownProps) {
```

Comment:
Do you need this? Maybe just pass the theme from the parent instead of connecting this component?

Reply:
Good point. I was just doing it the way `SpecialMentionItem` was being done (although theme was being passed, they still had a index.js) but you are right, no need to do that and I can just used the passed prop. Thanks :)

Reply:
Done!